### DISCUSSION TITLE
Is TIC-80 appropriate for College Level Educational Usage?

### DISCUSSION THREAD

**Discussion by gabemauger** [nesbox/TIC-80#1520] [Q&A]
Hello! I'm a research student at the New Mexico State University, and I've been analyzing the usage of different IDE's used in game-development related classes (Digital Game Design, Senior Projects, Intro to Game Design/Programming, etc.).

I came across TIC-80 after researching PICO-8, and I personally appreciate both of these software's for their simplistic approach.

However, I was wondering if any teachers, professors, or students have had any experiences with TIC-80 in the college classroom. What class was it used in, and how was it implemented?

If available, would you guys be willing to share some sample projects (while crediting the creator, of course)?

Thank you for sharing!

**Comment by Kouda-Ha** [comment]
Hello, I've just finished first year doing 'Computer Games Technology' at the Manchester Metropolitan University. Due to Covis-19 we did one module/course at a time and during the final module/course 'Computer Games Fundamentals' we used TIC-80 and Unity for our coursework/end of year project. 
This consisted of two choices, you either make a game in TIC-80, then make a terrain in Unity.
OR
Make a sprite sheet in TIC-80, then a 2D game in Unity.

I chose TIC-80 game, and Unity terrain. Here's a link to my game I uploaded to the TIC-80 website after everything was graded and etc:
[https://tic80.com/play?cart=1927](url)
tic80.com/play?cart=1927
The only difference between my project game and this upload to TIC-80 is I replaced my real name with my username I use online for general online anonymity.

The course itself didn't go into as much detail as the game ended up being but in the course which spanned 7 weeks we pretty much disected 'Hello World' default cart, played with making animations and making a 2D map, made a Pacman game, made a little spaceship shooting game with exploding animations on the meteors when hit, and our project was allowed to either be a completely brand new game from scratch or adding more things to these baseline games to make your own unique game.

It helped a lot as it was an easy enough thing to get used to, the drawing onto the physical map in TIC-80 personally helped me tremendously with making games, and all in all I think it is a really nice starter games engine to use for people who may have come from a none coding background, or who have done coding before.

I hope this comment helps and if I can be of any further use feel free to contact me, probably best on Twitter as I use it often @Kouda_Ha, not really checked Git in a while and I found this via TIC-80's tweet linking it.

**Reply by gabemauger** [reply]
Hello Kouda-Ha!

Thank you for your sharing your experience with it! Thank you especially for sharing your favorite aspects of the IDE; we are currently researching different tools we can implement in coursework, and it sounds like TIC-80 was very beneficial for you!

Also, thank you for the contact info, although it says I cannot message you? I'm still new to Twitter so I'm trying to figure everything out, lol.

Again, thank you for sharing!

**Reply by Kouda-Ha** [reply]
Oh damn, I haven't had to use git lately so didn't see this! Sorry about that, I just set it to allow messages from "everyone" on twitter, I didn't know it had a setting to stop people being able to DM message me directly, lol?
I'll still be generally available there if needed, but no worries either way as it's been two weeks so you've likely finished up a lot of things.
Yeah, it was pretty good and it is surprisingly fun to have a play with and see what people make with it.
